nix/deps/gradle/get_projects.sh

```diff
@@ -19,4 +19,4 @@ trap show_gradle_log ERR
 ./gradlew projects --no-daemon --console plain 2>&1 \
     | tee "${GRADLE_LOG_FILE}" \
     | grep "Project ':" \
-    | sed -E "s;^.--- Project '\:([@_a-zA-Z0-9\-]+)';\1;"
+    | sed -E "s;^.--- Project '\:([@_a-zA-Z0-9\-]+)'.*;\1;"
```

Comment:
Handles cases when the gradle project has a description, which shows up when running `gradle projects` as (`react-native-biometrics` - react-native-biometrics), breaking `make nix-update-gradle`. Here we're just adjusting the regex to ignore everything in the line after the closing (`).